initializers/sequelize.js

```diff
@@ -65,9 +63,10 @@ module.exports = {
           var nameParts = file.split("/");
           var name = nameParts[(nameParts.length - 1)].split(".")[0];
           var modelFunc = currySchemaFunc(require(dir + '/' + file));
-          api.models[name] = api.sequelize.sequelize.import(name, modelFunc);
+          api.sequelize.sequelize.import(name, modelFunc);
```

Comment:
In the sequelize `import` function, name is an unused argument - it would be good to remove its use here, and the parts that generate the `name` variable.